File: cpp/program/Baju.cpp
#include <iostream>
#include <string>
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris{
    private:
        string untuk, size, merk;
    
    public:
        Baju(){
            this->untuk = "";
            this->size = "";
            this->merk = "";
        }

        Baju(string untuk, string size, string merk){
            this->untuk = untuk;
            this->size = size;
            this->merk = merk;
        }

        // Getter untuk 'untuk'
        string getUntuk() {
            return untuk;
        }

        // Setter untuk 'untuk'
        void setUntuk(string untuk) {
            this->untuk = untuk;
        }

        // Getter untuk 'size'
        string getSize() {
            return size;
        }

        // Setter untuk 'size'
        void setSize(string size) {
            this->size = size;
        }

        // Getter untuk 'merk'
        string getMerk() {
            return merk;
        }

        // Setter untuk 'merk'
        void setMerk(string merk) {
            this->merk = merk;
        }
        
        // destruktor,
        ~Baju() {
        }
};

File: php/program/Baju.php
<?php
require('Aksesoris.php');

class Baju extends Aksesoris{
    private $untuk;
    private $size;
    private $merk;

    public function __construct($id, $nama, $stok, $harga, $urlfoto, $jenis, $bahan, $warna, $untuk, $size, $merk){
        parent::__construct($id, $nama, $stok, $harga, $urlfoto, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    // Getter dan Setter untuk untuk
    public function getUntuk(){
        return $this->untuk;
    }

    public function setUntuk($untuk){
        $this->untuk = $untuk;
    }

    // Getter dan Setter untuk size
    public function getSize(){
        return $this->size;
    }

    public function setSize($size){
        $this->size = $size;
    }

    // Getter dan Setter untuk merk
    public function getMerk(){
        return $this->merk;
    }

    public function setMerk($merk){
        $this->merk = $merk;
    }
}
?>